easybuild/easyblocks/p/python.py

```diff
@@ -109,6 +110,12 @@ def extra_options():
         extra_vars = {
             'ulimit_unlimited': [False, "Ensure stack size limit is set to '%s' during build" % UNLIMITED, CUSTOM],
             'ebpythonprefixes': [True, "Create sitecustomize.py and allow use of $EBPYTHONPREFIXES", CUSTOM],
+            'use_lto': [
+                None,
+                "Build with Link Time Optimization. Potentially unstable on some toolchains. Default: Auto-detect",
+                CUSTOM
+                ],
+            'optimized': [True, "Enable expensive, stable optimizations (PGO, etc)", CUSTOM],
```

Comment:
@Flamefire We should keep these ordered alphabetically, and I think they can be slightly clarified too, see https://github.com/Flamefire/easybuild-easyblocks/pull/1